src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -437,6 +402,67 @@ export class ExtensionEditor extends BaseEditor {
 		return super.setInput(input, options, token);
 	}
 
+	private setSubText(extension: IExtension, reloadAction: ReloadAction) {
+		if (reloadAction.enabled) {
```

Comment:
@sandy081 This `if` never returns true. This is because the update of the "enabled" status on the ReloadAction is not a synchronous operation. 

See https://github.com/Microsoft/vscode/blob/5ed816b5e74341adb7899a375acab09c182b44b3/src/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts#L1230

How do you suggest we tackle this?

Reply:
In such cases we listen to onDidChange event from the Action. See below

https://github.com/Microsoft/vscode/blob/c507f6347b3bc9961a23390e439c12128d690c82/src/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts#L464